### PR TITLE
osdocs11913rn: release note describing new behavior for crio wipe

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): 4.17

Issue: [OSDOCS-11913](https://issues.redhat.com//browse/OSDOCS-11913)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -558,6 +558,8 @@ For more information, see _Configuring Capacity Reservation by using machine set
 [id="ocp-4-17-nodes_{context}"]
 === Nodes
 
+Beginning in {product-title} 4.17, when a node is rebooted, the `crio wipe` command checks that the CRI-O binary exited cleanly. Those images that did not exit cleanly are targetted as corrupted and removed. This behavior prevents CRI-O from failing to start due to half-pulled images or other unsynced files. In {product-title} 4.15 and 4.16, the `crio wipe` command removed all images when a node was rebooted. The `crio wipe` command’s new behavior increases efficiency while still reducing the risk of image corruption when a node is rebooted.
```
Comment (cbippley) [inline release_notes/ocp-4-17-release-notes.adoc:561]:
```suggestion
Beginning in {product-title} 4.17, when a node is rebooted, the `crio wipe` command checks that the CRI-O binary exited cleanly. Those images that did not exit cleanly are targeted as corrupted and removed. This behavior prevents CRI-O from failing to start due to half-pulled images or other unsynced files. In {product-title} 4.15 and 4.16, the `crio wipe` command removed all images when a node was rebooted. The `crio wipe` command’s new behavior increases efficiency while still reducing the risk of image corruption when a node is rebooted.
```

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -558,6 +558,8 @@ For more information, see _Configuring Capacity Reservation by using machine set
 [id="ocp-4-17-nodes_{context}"]
 === Nodes
```
Comment (cbippley) [inline release_notes/ocp-4-17-release-notes.adoc:560]: This should have a heading. Example:
[id="ocp-4-17-enhancements-nodes-crio-wipe_{context}"]
==== New CRI-O command behavior

**Review by cbippley — COMMENTED** [review]
One nit and and you need a heading. Otherwise, LGTM.

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (DCChadwick) [reply release_notes/ocp-4-17-release-notes.adoc:561]: Ha! Thanks for catching that.

**Comment by DCChadwick** [comment]
@sunilcio This is ready for QE review

File: release_notes/ocp-4-17-release-notes.adoc (hunk 2 above)
Comment (DCChadwick) [reply release_notes/ocp-4-17-release-notes.adoc:560]: Thanks! I've added a heading.

File: release_notes/ocp-4-17-release-notes.adoc [hunk 3]
```diff
@@ -556,7 +556,9 @@ Node disruption policies for Google Cloud Platform (GCP) clusters has been promo
 For more information, see _Configuring Capacity Reservation by using machine sets_ for xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-capacity-reservation_creating-machineset-azure[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure.adoc#machineset-azure-capacity-reservation_cpmso-config-options-azure[control plane] machine sets.
 
 [id="ocp-4-17-nodes_{context}"]
-=== Nodes
+=== New CRIO command behavior
```
Comment (cbippley) [inline release_notes/ocp-4-17-release-notes.adoc:559]:
```suggestion
[id="ocp-4-17-enhancements-nodes-crio-wipe_{context}"]
==== New CRI-O command behavior
```
The heading should look like this.

Comment (DCChadwick) [reply release_notes/ocp-4-17-release-notes.adoc:559]: Oh, thank you!

Comment (DCChadwick) [reply release_notes/ocp-4-17-release-notes.adoc]: Done! Thank you!

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**